add info about rate limits in agent-api (#730)
livechat/livechat-public-docs: content/messaging/agent-chat-api/index.mdx

Summing up, if the user token starts with `fra-`, you should add the `X-Region: fra` header. If the token starts with `dal-`, you don’t have to specify the header.

## Rate limits

> To prevent sending a massive number of requests, there are rate limits for methods' calls. After exceeding them, the requester will get a `Too many requests` error in a response to a specific request. After some time, the request will be unblocked so it can be sent again.

Reviewer: ```suggestion
To prevent sending a massive number of requests, there are rate limits for method calls. After exceeding those limits, the requester will get a `Too many requests` error in the response of a specific request. After some time, the request will be unblocked so it can be sent again.
```

Reviewer: Shouldn't we specify, at least approximately, after how long the request will be unblocked?

Author: Unfortunately, we cannot specify that yet, these times can be different for each rate limit.

| `license_not_found`         | License not found             | License with the specified ID doesn't exist.                                         |
| `misdirected_request`**\*** | Wrong region                  | Client's request should be performed to another region.                              |
| `request_timeout`           | Request timed out             | Timeout threshold is 15 seconds.                                                     |
| `unsupported_version`       | Unsupported version           | Unsupported protocol version.                                                        |
| `validation`                | Wrong format of request       |                                                                                      |
| `wrong_product_version`     | Wrong product version         | License is not LiveChat 3 (probably still LiveChat 2).                               |
| `chat_inactive`             | No active chat thread         | An action that requires an active thread performed on a chat with no active threads. |

> | `too_many_requests`         | Too many requests             | A request has been rate limited; after some time it will be unblocked automatically  |

Reviewer: ```suggestion
| `too_many_requests`         | Too many requests             | The request's rate limit was exceeded. It'll be unblocked automatically after some time.  |
```
